billing-api/billing_server/billing/config/default.py

```diff
@@ -13,36 +13,26 @@
 # OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER
 # IN CONTRACT, STRICT LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN
 # ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
-DEBUG = True  # Debug mode for flask
-SECRET_KEY = 'random, secret, super duper secret key'
-AUTH_URI = 'http://142.1.177.54:5000/v2.0'  # Keystone/Identity API endpoint
-INVOICE_API = 'http://localhost:4000/invoice'
-MYSQL_URI = 'mysql://root:test@142.1.177.54:3306'  # Mysql URI
-VALID_BUCKET_SIZES = ['daily', 'weekly', 'monthly', 'yearly']  # Bucketing options for query.
-#FLASK_LOG_FILE = '/srv/billing-api/logs/billing.log'
-FLASK_LOG_FILE = './logs/billing.log'
-BILLING_ROLE = 'billing_test'
-#INVOICE_ROLE = 'invoice'
-INVOICE_ROLE = 'invoice_test'
-OICR_ADMIN = 'oicr_admin'
+import json
+
+config={}
+with open('../globalConfig.json') as json_file: # This file is under billing/ folder
```

Comment:
did you confer with @jmimico that he overwrites this default.py file when running the docker image? Otherwise this won't work.